master/buildbot/statistics/storage_backends.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
```

Comment:
I'd suggest to make this file `buildbot/statistics/storage_backends/base.py` and move InfluxDB related code to `buildbot/statistics/storage_backends/influxdb.py` as we usually do.